README.md

```diff
@@ -5,7 +5,7 @@ Write down two truths and one lie below.
 
 Once you're done, submit a pull request (do NOT commit directly to the main branch)! We'll try to guess the lie before we merge your branch with the main branch.
 
-YOUR NAME HERE:
-1. A truth (or a lie)
-2. A lie (or a truth)
-3. Another truth (or your lie)
+YOUR NAME HERE: alice
+1. my favorite color is green
```

Comment:
this might be a bait, but I'm gonna go with only one layer of deception and say that the simplest statement is a lie